packages/eslint-plugin-fxa/package.json

```diff
@@ -12,7 +12,7 @@
     "ramda": "^0.26.1"
   },
   "devDependencies": {
-    "mocha": "6.2.2"
+    "mocha": "10.2.0"
```

Comment:
Just double checking that we want 10.2.0 and not ^10.2.0. Most of our package.json ref mocha as ^10.0.0.